helm/svc-bie-kafka/values.yaml

```diff
@@ -24,7 +24,7 @@ ports:
 
 livenessProbe:
   httpGet:
-    path: /actuator/health/liveness
+    path: /actuator/health/liveness  
```

Comment:
is the empty space and intended change here?

Reply:
No that wasn't intended